### ISSUE TITLE
.HEIC Images Uploaded from iPhone Not Displaying on Mac App or Web Browser

### ISSUE THREAD

**Issue by juliengonthier** [RocketChat/Rocket.Chat#30586]
When I upload a photo taken with my iPhone (in .heic format) using the RocketChat app for iPhone, it works perfectly. However, this same photo isn't visible in the RocketChat app on the Mac or in the web browser. I am using the latest version of the app.

<img width="318" alt="Screenshot 2023-10-06 at 10 52 53" src="https://github.com/RocketChat/Rocket.Chat/assets/18209167/023005cc-f5a7-4cde-b438-4b4748020bb1">

**Comment by YashGupt29** [comment]
maybe Rocket chat does not support it in heic format convert it into jpg from an online convertor and then try!

**Comment by juliengonthier** [comment]
> maybe Rocket chat does not support it in heic format convert it into jpg from an online convertor and then try!

Of course, it works with a JPEG file. However, .HEIC is the standard format on iPhones, so I believe the RocketChat app for iPhone should support this format...

**Comment by noobbbbb** [comment]
duplicate
#23638

**Comment by limer-org** [comment]
w moim przypadku pomogło podpięcie pod `/opt/Rocket.Chat/programs/server/npm/node_modules/@img/sharp-libvips-linux-x64/lib/libvips-cpp.so.42` własnej skompilowanej biblioteki libvips 8.17.2 opartej o skompilowaną, pełną wsparcia HEIC/HEIF bibliotekę libheif, w wersji 1.20.2. Debian 13. RocketChat 7.10.
